src/ui/screens/EventsCalendarTeaser.tsx

```diff
@@ -0,0 +1,60 @@
+import {LoadingBox} from '@ui/components/LoadingBox';
+import {SectionTeaserContainer} from '@ui/components/SectionTeaserContainer';
+import {useCalendarEvents} from 'src/api/hooks/useCalendarEvents';
+import {View, FlatList, StyleSheet} from 'react-native';
+import React from 'react';
+import {Divider, Text, useTheme} from '@ui/library';
+import {standardPadding} from '@ui/styles';
+
+type Props = {
+  onPress: () => void;
+};
+
+export function EventsCalendarTeaser(props: Props) {
+  const {colors} = useTheme();
+  const {data: events, loading} = useCalendarEvents();
+  const topThreeEvents = events?.slice(0, 3);
+  const formateDate = (date: string) => {
+    const formatedDate = new Date(date);
+    const [eventDate, eventMonth] = [formatedDate.getDate(), formatedDate.getMonth()];
+    return eventDate.toString().concat('/').concat(eventMonth.toString());
+  };
```

Comment:
Independent util methods like this can be outside the component so that these are not affected by component re-renders.

But in this case we should actually make use of the `dayjs` util and the method should live in `utils/date.ts`